proposals/description/fip_70.ts

```diff
@@ -0,0 +1,52 @@
+import { ProposalDescription } from '@custom-types/types';
+
+const fip_70: ProposalDescription = {
+  title: 'FIP-70: ETH/FEI Liquidity migration to Balancer',
+  commands: [
+    {
+      target: 'core',
+      values: '0',
+      method: 'grantMinter(address)',
+      arguments: ['{balancerDepositFeiWeth}'],
+      description: 'Grant FEI/WETH Pool deposit the FEI_MINTER_ROLE role'
+    },
+    {
+      target: 'collateralizationOracle',
+      values: '0',
+      method: 'addDeposits(address[])',
+      arguments: [['{balancerDepositFeiWeth}']],
+      description: 'Add Balancer WETH/FEI deposit to CR oracle'
+    },
+    {
+      target: 'uniswapPCVDeposit',
+      values: '0',
+      method: 'withdraw(address,uint256)',
+      arguments: ['{balancerDepositFeiWeth}', '20000000000000000000000'],
```

Comment:
we should do withdrawRatio, 50% no?

Reply:
I think there is even a WETH wrapping utility

Reply:
will do this